src/js/ripple/bignumber.js

```diff
@@ -0,0 +1,28 @@
+var BigNumber = require('bignumber.js');
+var extend = require('extend');
+
+function BigNumberWrapper(value, base) {
+  // reset config every time a BigNumber is instantiated so that
```

Comment:
Makes sense, this all seems a bit naff. Shame bignumber doesn't have a factory to create isolated contexts.